share/xml/csg_defaults.xml.in

```diff
@@ -263,12 +263,6 @@
       <default_reg>0
         <DESC> default magnitude for regularization parameter if not given for the group explicitly, default =0</DESC>
       </default_reg>
-      <bonded_method>ibi
-        <DESC> method for calculating updates on the bonded interactions. ibi or imc (not implemented), default=ibi </DESC>
-      </bonded_method>
-      <settings_nonbonded>imc
-        <DESC> settings file to be used with csg_stat --do-imc. only needed if bonded_method=ibi</DESC>
-      </settings_nonbonded>
```

Comment:
We might want to add these two options to the check for obsolete options.

Or maybe not was it hasn't been in any release.

Reply:
Then let's not.